src/client/components/MusicReserve/AlbumTable.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View } from '@instructure/ui-view';
+import { Text } from '@instructure/ui-text';
+import { CondensedButton } from '@instructure/ui-buttons';
+import { Table } from '@instructure/ui-table';
+
+export const AlbumTable = ({ allAlbums, handleClick }) => {
+  AlbumTable.propTypes = {
+    allAlbums: PropTypes.array,
+    handleClick: PropTypes.func,
+  };
+
+  return (
+    <Table caption="Albums">
+      <Table.Head>
+        <Table.Row>
+          <Table.ColHeader id="title">Title</Table.ColHeader>
+          <Table.ColHeader id="count">Number of tracks</Table.ColHeader>
+          <Table.ColHeader id="meta">Descriptions</Table.ColHeader>
+        </Table.Row>
+      </Table.Head>
+      <Table.Body>
+        {allAlbums.map(album => (
+          <Table.Row>
```

Comment:
When mapping an array, does React give a warning in the browser console that each <Table.Row> needs a unique key prop? I've encountered that warning when mapping arrays to UI before